sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -787,7 +787,7 @@ def get_table(self, project_id, dataset_id, table_id):
       HttpError: if lookup failed.
     """
     request = bigquery.BigqueryTablesGetRequest(
-        projectId=project_id, datasetId=dataset_id, tableId=table_id)
+        projectId=project_id, datasetId=dataset_id, tableId=table_id, table_view=table_view)
```

Comment:
Check https://cloud.google.com/bigquery/docs/reference/rest/v2/tables/get. `table_view` is not correct.